yt_dlp/extractor/radiocomercial.py

```diff
@@ -0,0 +1,142 @@
+import functools
+
+from .common import InfoExtractor
+from ..utils import (
+    OnDemandPagedList,
+    extract_attributes,
+    get_element_by_class,
+    get_element_html_by_class,
+    get_element_text_and_html_by_tag,
+    get_elements_html_by_class,
+    int_or_none,
+    join_nonempty,
+    try_call,
+    unified_strdate,
+)
+from ..utils.traversal import traverse_obj
+
+
+class RadioComercialIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?radiocomercial\.pt/podcasts/[^/]+/\D*(?P<season>\d+)/(?P<id>[\w-]+)'
+    _TESTS = [
+        {
+            'url': 'https://radiocomercial.pt/podcasts/o-homem-que-mordeu-o-cao/t6/taylor-swift-entranhando-se-que-nem-uma-espada-no-ventre-dos-fas',
+            'md5': '5f4fe8e485b29d2e8fd495605bc2c7e4',
+            'info_dict': {
+                'id': 'taylor-swift-entranhando-se-que-nem-uma-espada-no-ventre-dos-fas',
+                'ext': 'mp3',
+                'title': 'Taylor Swift entranhando-se que nem uma espada no ventre dos fãs.',
+                'description': None,
+                'release_date': '20231025',
+                'thumbnail': r're:https://radiocomercial.pt/upload/[^.]+.jpg',
+                'season': 6
+            }
+        },
+        {
+            'url': 'https://radiocomercial.pt/podcasts/convenca-me-num-minuto/t3/convenca-me-num-minuto-que-os-lobisomens-existem',
+            'md5': '47e96c273aef96a8eb160cd6cf46d782',
+            'info_dict': {
+                'id': 'convenca-me-num-minuto-que-os-lobisomens-existem',
+                'ext': 'mp3',
+                'title': 'Convença-me num minuto que os lobisomens existem',
+                'description': None,
+                'release_date': '20231026',
+                'thumbnail': r're:https://radiocomercial.pt/upload/[^.]+.jpg',
+                'season': 3
+            }
+        },
+        {
+            'url': 'https://radiocomercial.pt/podcasts/inacreditavel-by-ines-castel-branco/t2/o-desastre-de-aviao',
+            'md5': '69be64255420fec23b7259955d771e54',
+            'info_dict': {
+                'id': 'o-desastre-de-aviao',
+                'ext': 'mp3',
+                'title': 'O desastre de avião',
+                'description': 'md5:8a82beeb372641614772baab7246245f',
+                'release_date': '20231101',
+                'thumbnail': r're:https://radiocomercial.pt/upload/[^.]+.jpg',
+                'season': 2
+            },
+            'skip': 'inconsistent md5',
```

Comment:
Actually sorry, this would be better to still check if the metadata is correct:
```suggestion
            'params': {
                # inconsistant md5
                'skip_download': True,
            },
```